apps/explorer/locales/en-US/translation.json

```diff
@@ -1,7 +1,8 @@
 {
   "ALPH balance": "ALPH balance",
   "Address group": "Address group",
-  "Address": "Address",
+  "Addresses_one": "Address",
+  "Addresses_other": "Addresses",
```

Comment:
Why using this syntax over "Address" / "Addresses"?

Reply:
Check the docs for i18next regarding plurals: https://www.i18next.com/translation-function/plurals

Reply:
Okay, thanks for the link. Let's hope translators understand this.

Reply:
They don't need to if they are going through Crowdin.I think it displays it accordingly. Let's see